src/content/configuration/dev-server.md

```diff
@@ -930,7 +930,7 @@ module.exports = {
 };
 ```
 
-The origin of the host header is kept when proxying by default, you can set `changeOrigin` to `true` to override this behaviour. It is useful in some cases like using [name-based virtual hosted sites](https://en.wikipedia.org/wiki/Virtual_hosting#Name-based).
+The origin of the host header is kept when proxying by default, you can set `changeOrigin` to `true` to override this behavior. It is useful in some cases like using [name-based virtual hosted sites](https://en.wikipedia.org/wiki/Virtual_hosting#Name-based).
```

Comment:
Please revert. We can endlessly receive change proposals for american vs british english e.g. colour and color etc.

Reply:
reverted